arm/Microsoft.RecoveryServices/vaults/backupStorageConfig/deploy.bicep

```diff
@@ -0,0 +1,34 @@
+@description('Required. Name of the Azure Recovery Service Vault')
+@minLength(1)
+param recoveryVaultName string
+
+@description('Optional. Change Vault Storage Type (Works if vault has not registered any backup instance)')
+@allowed([
+  'GeoRedundant'
+  'LocallyRedundant'
+  'ReadAccessGeoZoneRedundant'
+  'ZoneRedundant'
+])
+param vaultStorageType string = 'GeoRedundant'
```

Comment:
Please replace with `storageModelType` to reflect official arm reference

Reply:
Updated!